### DISCUSSION TITLE
Apache 衡量项目成熟度的方式

### DISCUSSION THREAD

**Discussion by tisonkun** [korandoru/town#15] [Comments]
https://town.korandoru.io/communities/apache-maturity-model

**Comment by WillemJiang** [comment]
ambari 项目已经复活， 可以刷新一下案例。

**Reply by tisonkun** [reply]
贴下对应的邮件列表链接呗，我今天更新下。

**Reply by WillemJiang** [reply]
在general@incubator.a.o 上面有讨论 https://lists.apache.org/thread/9yq7sw5scovdzo7bs4clzcm2447plf18

**Reply by WillemJiang** [reply]
相关的讨论历史 https://lists.apache.org/thread/jk5663pol2f078t5xxvz73hy4mmq5zfs

**Reply by tisonkun** [reply]
感谢，案例已更新。

**Comment by pacoxu** [comment]
这个是不是也可以看作是一种 开源软件采纳使用的 评估手册？

**Reply by tisonkun** [reply]
可以的。不过这个模式主要视角是社群维护者审视自己，对于采用方来说，有些内容并不那么重要。
